src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -66,8 +66,13 @@ export default function CDCComparisonContainer() {
           sample.
         </p>
       </div>
+      <h3>DAILY NEW TESTS IN THE U.S.</h3>
       <div style={{ display: 'flex' }}>
         <div style={{ flexGrow: 1, width: '50%' }}>
+          <div>
+            <h4 style={{ marginBottom: '0.02rem' }}>C.D.C.</h4>
```

Comment:
Let's avoid inline styles in favor of classes (preferably in the new `dashboard.module.scss`).